examples/streaming/kafka/src/test/scala/org/apache/gearpump/streaming/examples/kafka/KafkaStreamProducerSpec.scala

```diff
@@ -87,7 +90,7 @@ class KafkaStreamProducerSpec extends PropSpec with Matchers with BeforeAndAfter
       producer.onNext(Message("next"))
 
       var expectedMessage = messages
-      verify(context).output(argMatch[Message](msg => {
+      verify(context, new After(10000L, VerificationModeFactory.times(1))).output(argMatch[Message](msg => {
```

Comment:
@kkasravi 

I know there are ocassially timeout in UT, but making the timeout to """10000L""" will possibly hang the UT?